specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-02-preview/managedClusters.json

```diff
@@ -6979,6 +7021,91 @@
       "required": [
         "enabled"
       ]
+    },
+    "GuardrailsProfile": {
+      "type": "object",
+      "properties": {
+        "systemExcludedNamespaces": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of namespaces specified by AKS to be excluded from Guardrails",
+          "items": {
+            "type": "string"
+          }
+        },
+        "version": {
+          "type": "string",
+          "description": "The version of constraints to use"
+        },
+        "level": {
+          "type": "string",
+          "enum": [
+            "Off",
+            "Warning",
+            "Enforcement"
+          ],
+          "x-ms-enum": {
+            "name": "level",
+            "modelAsString": true
+          },
+          "description": "The guardrails level to be used"
+        },
+        "includedNamespaces": {
+          "description": "Customer-specified list of namespaces that must be included in guardrails",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "excludedNamespaces": {
+          "description": "Customer-specified list of namespaces that must be excluded from guardrails",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "level",
+        "version"
+      ],
+      "description": "The Guardrails profile."
+    },
+    "ListGuardrailsVersionsResponse": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "readOnly": true,
+          "type": "string"
+        },
+        "isPreview": {
+          "type": "boolean",
+          "description": "Describes whether this version is in preview"
+        },
+        "default": {
+          "type": "boolean",
+          "description": "Describes whether this version is default"
+        },
+        "data": {
```

Comment:
From [CI](https://github.com/Azure/azure-rest-api-specs/pull/20171/checks?check_run_id=8025953973), missing identifier for this array.

May add`"x-ms-identifiers": [],` to bypass the check.